packages/istanbul-lib-report/test/tree.test.js

```diff
@@ -9,31 +8,32 @@ const BaseNode = t.Node;
 const Visitor = t.Visitor;
 const CompositeVisitor = t.CompositeVisitor;
 
-function TestNode(name) {
-    BaseNode.call(this);
-    this.name = name;
-    this.parent = null;
-    this.children = [];
-}
+class TestNode extends BaseNode {
```

Comment:
This had to be converted because it's no longer valid to run `BaseNode.call(this)` as BaseNode is a class (so it must be called with `new`).  This is not a breaking change as `BaseNode` is not exposed outside this module (`lib/tree.js` is not part of the public API).